src/providers/ad/ad_common.c

```diff
@@ -1091,7 +1091,9 @@ ad_resolve_callback(void *private_data, struct fo_server *server)
                                                  ad_krb5info_file_filter);
         if (ret != EOK) {
             DEBUG(SSSDBG_MINOR_FAILURE,
-                  "write_krb5info_file failed, authentication might fail.\n");
+                  "write to %s/kdcinfo.%s failed, authentication might fail.\n",
+                  PUBCONF_PATH,
+                  service->krb5_service->realm);
```

Comment:
Put those arguments on one line if it fits 80-chars limits, which I think does. I.e.
```
            DEBUG(SSSDBG_MINOR_FAILURE,
                  "write to %s/kdcinfo.%s failed, authentication might fail.\n",
                  PUBCONF_PATH, service->krb5_service->realm);
```

Please do it in all messages in this patch.